src/ui_utils.c

```diff
@@ -2477,10 +2477,23 @@ void ui_init_builder(void)
 static void init_custom_style(void)
 {
 #if GTK_CHECK_VERSION(3, 0, 0)
-	gchar *css_file = g_build_filename(app->datadir, "geany.css", NULL);
+	const struct {
+		guint version;
+		const gchar *file;
+	} css_files[] = {
+		{ 20, "geany-3.20.css" },
+		{ 0, "geany-3.0.css" },
+	};
+	guint gtk_version = gtk_get_minor_version();
+	gsize i;
+	gchar *css_file;
 	GtkCssProvider *css = gtk_css_provider_new();
 	GError *error = NULL;
 
+	for ( i = 0 ; i < G_N_ELEMENTS(css_files) && css_files[i].version > gtk_version ; ++i )
+	{}
```

Comment:
Not a huge deal, but this `for` loop and the above inline struct/variable don't match [Geany's coding style](https://github.com/geany/geany/blob/1.27.0/HACKING#L228). No spaces inside parens or before `;`, braces on newlines, etc).